### PR TITLE
vendor: update bitbox02-api-go

### PR DESCRIPTION
**Pull request by benma**
```
go get github.com/digitalbitbox/bitbox02-api-go@935daea2c32724eb105509d4730926cf69a6d9dd
go mod tidy
make go-vendor
```

This activates the antiklepto protocol for BTC signing for the BitBox02 v9.4.0+.

### CODE REVIEW

**Review by x1ddos — APPROVED** [review]
LGTM

File: vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+// antikleptoVerify verifies that hostNonce was used to tweak the nonce during signature
+// generation according to k' = k + H(clientCommitment, hostNonce) by checking that
+// k'*G = signerCommitment + H(signerCommitment, hostNonce)*G.
+func antikleptoVerify(hostNonce, signerCommitment, signature []byte) error {
+	signerCommitmentPubkey, err := btcec.ParsePubKey(signerCommitment, btcec.S256())
+	fmt.Println("LOL===")
+	fmt.Println(hex.EncodeToString(hostNonce))
+	fmt.Println(hex.EncodeToString(signerCommitment))
+	fmt.Println(hex.EncodeToString(signature))
```
Comment (benma) [inline vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go:48]: OOOPS :joy:

Comment (benma) [reply vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go:48]: https://github.com/digitalbitbox/bitbox02-api-go/pull/49

Comment (x1ddos) [reply vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go:48]: wait, I'm pretty sure I didn't see any lols in https://github.com/digitalbitbox/bitbox02-api-go/pull/48... how did it get there? :see_no_evil:

Comment (benma) [reply vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go]: i added a unit test as you requested, and then merged it without re-requesting a review. serves me right! :sweat_smile:

Comment (x1ddos) [reply vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go]: maybe we should add a "lol" linter. THAT would be one linter I'd never complain about :rofl:

Comment (benma) [reply vendor/github.com/digitalbitbox/bitbox02-api-go/api/firmware/secp256k1.go]: That would be so helpful! Was not the first and probably not the last time ;)

**Pull request merged**